app/routes/people/person/personal-information.js

```diff
@@ -11,11 +17,99 @@ export default class PeoplePersonPersonalInformationRoute extends Route {
       include: [
         'mandatories.mandate.governing-body.is-time-specialization-of.administrative-unit.classification',
         'mandatories.contacts',
+        'agents-in-position',
```

Comment:
I would include all the related relationships already. At the moment this will trigger separate calls for each position in the list.

Reply:
> I would include all the needed relationships already. At the moment this will trigger separate calls for each position in the list when accessing relationship data.

I included the missing one but for some reason I still have a call to contacts :/